server/bleep/src/webserver/repos.rs

```diff
@@ -33,7 +33,7 @@ impl From<(&RepoRef, &Repository)> for Repo {
     fn from((key, repo): (&RepoRef, &Repository)) -> Self {
         Repo {
             provider: key.backend(),
-            name: key.display_name(),
+            name: key.to_string(),
```

Comment:
This is not an equivalent change, look at the implementation of `display_name()`.

Reply:
Good spot!